content/en/docs/reference/kubernetes-api/labels-annotations-taints.md

```diff
@@ -114,3 +116,151 @@ The scheduler (through the _VolumeZonePredicate_ predicate) also will ensure tha
 
+## node.kubernetes.io/windows-build {#nodekubernetesiowindows-build}
+
+Example: `node.kubernetes.io/windows-build=10.0.17763`
+
+Used on: Node
+
+Starting in v1.17, this label is automatically added to adapt to the situation that the same cluster uses multiple versions of windows server.
+
+The label's value is in the format "MajorVersion.MinorVersion.BuildNumber".
+
+See [node.kubernetes.io/windows-build](#nodekubernetesiowindows-build).
+
+## service.kubernetes.io/headless {#servicekubernetesioheadless}
+
+Example: `service.kubernetes.io/headless=""`
+
+Used on: Service
+
+The label is added by Controller to an Endpoint denoting if its parent Service is Headless.
+
+See [service.kubernetes.io/headless](#servicekubernetesioheadless).
+
+## kubernetes.io/service-name {#kubernetesioservice-name}
+
+Example: `kubernetes.io/service-name="nginx"`
+
+Used on: Service
+
+The label is the tag name we use to differentiate multiple services. Used currently for ELBs only.
+
+See [kubernetes.io/service-name](#kubernetesioservice-name).
+
+## endpointslice.kubernetes.io/managed-by {#endpointslicekubernetesiomanaged-by}
+
+Example: `endpointslice.kubernetes.io/managed-by="controller"`
+
+Used on: Endpoint
+
+The label is used to indicate the controller or entity that manages an EndpointSlice. This label aims to enable different EndpointSlice objects to be managed by different controllers or entities within the same cluster.
+
+When upgrading to v1.17 with a cluster with EndpointSlice enabled, the `endpointslice.kubernetes.io/managed-by` label needs to be set on each EndpointSlice.
+
+See [endpointslice.kubernetes.io/managed-by](#endpointslicekubernetesiomanaged-by).
+
+## endpointslice.kubernetes.io/skip-mirror {#endpointslicekubernetesioskip-mirror}
+
+Example: `endpointslice.kubernetes.io/skip-mirror="true"`
+
+Used on: Endpoint
+
+The label can be set to true on an Endpoint resource to indicate that the EndpointSliceMirroring controller should not mirror this resource with EndpointSlices.
+
+See [endpointslice.kubernetes.io/skip-mirror](#endpointslicekubernetesioskip-mirror).
+
+## service.kubernetes.io/service-proxy-name {#servicekubernetesioservice-proxy-name}
+
+Example: `service.kubernetes.io/service-proxy-name="foo-bar"`
+
+Used on: Service
+
+The label indicates that a custom service proxy will implement this service, kube-proxy does not generate iptables for the service.
+
+See [service.kubernetes.io/service-proxy-name](#servicekubernetesioservice-proxy-name).
+
+# Annotations
+
+## experimental.windows.kubernetes.io/isolation-type
+
+Example: `experimental.windows.kubernetes.io/isolation-type: "hyperv"`
+
+The annotation is used to run windows containers with hyperv isolation.
```

Comment:
```suggestion
The annotation is used to run Windows containers with Hyper-V isolation.
```

Reply:
I presume this is an annotation for a Pod. Is it a hint or something stronger? What acts on it (eg the kubelet)?